propulse_web/urls.py

```diff
@@ -30,10 +30,10 @@
     path('', home_view, name='home'),
     path('about/', TemplateView.as_view(template_name='about.html'), name='about'),
     path('join/', TemplateView.as_view(template_name='join.html'), name='join'),
-
-
+    path('sponsor', TemplateView.as_view(template_name='sponsor.html'), name='sponsor'),
     # Includes urls from app
     path('blog/', include('blog.urls')),
+    path(r'^ckeditor/', include('ckeditor_uploader.urls')),
```

Comment:
r'something/' is depreciated syntax
```suggestion
    path('ckeditor/', include('ckeditor_uploader.urls')),
```